os/drivers/video/Kconfig

```diff
@@ -29,4 +29,91 @@ config VIDEO_SOURCE
 config VIDEO_NULL
 	bool "Driver for Dummy Video lowerhalf"
 	depends on VIDEO_SOURCE
-	default n
\ No newline at end of file
+	default n
+
+if VIDEO_NULL
+
+	comment "Select from the following supported options"
+
+choice
+
+	prompt "Supported video-frame formats"
+
+config VIDEO_FORMAT_MJPEG
+	bool "MJPEG Video Format"
+	depends on VIDEO_NULL && !VIDEO_FORMAT_YUY2
+	default n
+
+config VIDEO_FORMAT_YUY2
+	bool "YUY2 Video Format"
+	depends on VIDEO_NULL && !VIDEO_FORMAT_MJPEG
+	default n
+
+endchoice
+
+choice
+        prompt   "Supported frame reolutions (-U)"
+
+config QVGA
+        bool "(0)QVGA (320x240)"
+        default n
+
+config VGA
+        bool "(1)VGA (640x480)"
+        default n
+
+config QUADVGA
+        bool "(2)QUADVGA (1280x960)"
+        default n
+
+config HD
+        bool "(3)HD (1280x720)"
+        default n
+
+comment "Higher frame resolutions may not be supported"
+comment " due to device memory restrictions"
+
+config FHD
+        bool "(4)FHD (1920x1080)"
+        default n
+
+config 5M
+        bool "(5)5M (2560x1920)"
+        default n
+
+config 3M
+        bool "(6)3M (2048x1536)"
+        default n
+
+endchoice
+
+choice
+        prompt   "Supported frame rates (-K)"
+
+config FR_30
+        bool "(0)1/30"
+        default n
+
+config FR_20
+        bool "(1)1/20"
+        default n
+
+config FR_15
+        bool "(2)1/15"
+        default n
+
+config FR_10
+        bool "(3)1/10"
+        default n
+
+endchoice
+
+config NUM_FRAMES
+	int "No. of frames to be rendered"
```

Comment:
do we have any range or limit for this setting

Reply:
Not as such.

Reply:
how will it validate data? is it handled at source code?

Reply:
These Kconfig additions have only been done for reference to the NULL Driver user.
The final options for the NULL Driver are still to be provided with the camera example command from tash.
Mentioned at line 115 of Kconfig.